.github/workflows/cla-check.yml

```diff
@@ -0,0 +1,12 @@
+name: cla-check
```

Comment:
The names used across the PR seem very inconsistent. We have:
```
name: cla-check
name: Check if CLA signed
name: Go linters
name: Check code formatting
name: Golang Security Checker
name: Run Gosec Security Scanner
name: Upload SARIF file
```

This needs to consider closely the convention already in place for existing workflows here and in Pebble, and then try to be clean and consistent.

Reply:
Updated in https://github.com/canonical/chisel/pull/31/commits/c013935a13f0b2c47203777501ea88610b703d3f. Thanks!